src/k8s/pkg/k8sd/features/metrics-server/metrics_server_test.go

```diff
@@ -1,12 +1,12 @@
-package features_test
+package metrics_server_test
 
 import (
 	"context"
 	"testing"
 
 	"github.com/canonical/k8s/pkg/client/helm"
 	helmmock "github.com/canonical/k8s/pkg/client/helm/mock"
-	"github.com/canonical/k8s/pkg/k8sd/features"
+	metrics_server "github.com/canonical/k8s/pkg/k8sd/features/metrics-server"
```

Comment:
The package is called `metrics_server`. How does this work with `-`?

Reply:
files in `pkg/my-package` get a `package my_package` header, and then Go imports it as `my_package ".../pkg/my-package"`